setup.py

```diff
@@ -46,12 +46,19 @@
   keywords='appscale google-app-engine python java go php',
   platforms='Posix; MacOS X',
   install_requires=[
+    'adal',
+    'azure==2.0.0rc5',
+    'cryptography',
     'argparse',
     'boto',
     'google-api-python-client>=1.5.0',
+    'haikunator',
     'httplib2',
+    'jwt',
```

Comment:
I noticed that we don't seem to import from this or cryptography directly. Are these dependencies of other dependencies (ie will another package install them if these lines are removed), or do they need to be defined explicitly here?